docs/docsgen/source/index.rst

```diff
@@ -0,0 +1,19 @@
+
+.. _l-main-doc-page:
+
+Python Documentation of ONNX
```

Comment:
May be change title to `Documentation of ONNX Python Package` ?

Reply:
done